### DISCUSSION TITLE
Rename file by first creating a folder to place it within

### DISCUSSION THREAD

**Discussion by reagle** [beetbox/beets#4613] [Q&A] [answered]
I'd like to follow the [Plex naming standard](https://support.plex.tv/articles/200265296-adding-music-media-from-folders/) and rename an MP3 by first moving it to a folder, e.g.,: `%{artist} - %{album}` and within that artist/album folder have the name `%{track} %{title}` . Is this possible? If so, how? Thanks!

**Comment by wisp3rwind** [comment, marked as answer]
That should be possible using [path formats](https://beets.readthedocs.io/en/stable/reference/pathformat.html).
